packages/paste-website/src/pages/components/paragraph/index.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Paragraph
+package: '@twilio-paste/paragraph'
+description: The paragraph component is used to render blocks of text, with a preset amount of space beneath it to distinguish from adjacent content.
+---
+
+import {graphql} from 'gatsby';
+import Img from 'gatsby-image';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/paragraph/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/paragraph"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/paragraph/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/paragraph"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-anchor--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<LivePreview scope={{Paragraph}} language="jsx">
+  {`<Paragraph>
+  Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+</Paragraph>`}
+</LivePreview>
+
+## Guidelines
+
+By default, the paragraph component should provide consistent values for font-size, line-height, and margin-bottom across themes.
+
+### About Paragraphs
+
+Paragraphs are distinct sections of content, typically dealing with a single theme, and are indicated by a new line. Paragraphs are often used for a majority of text on the page and should be able to support a variety of information-dense layouts.
+
+Paragraph is part of the Typography component, which includes Header and List as well.
```

Comment:
Typography component -> Typography package
Would be nice if we could link to it too (on NPM or docs when we eventually have them).